src/security/builtin_plugins/tests/CMakeLists.txt

```diff
@@ -59,7 +59,16 @@ set(security_crypto_test_sources
     "set_remote_participant_crypto_tokens/src/set_remote_participant_crypto_tokens_utests.c"
 )
 
-add_cunit_executable(cunit_security_plugins ${security_auth_test_sources} ${security_ac_test_sources} ${security_crypto_test_sources})
+add_cunit_executable(cunit_security_plugins
+  ${security_auth_test_sources}
+  ${security_ac_test_sources}
+  ${security_crypto_test_sources}
+  ${CYCLONEDDS_OPENSSL_APPLINK})
```

Comment:
With this it will "parse" the source for CUnit test signatures. It'll not be a problem in practice, but you could opt to add the source file to the target afterwards.